apiv2/views.py

```diff
@@ -702,7 +702,9 @@ def get(self, request,  *args, **kwargs):
                 msg='Sounds in pack %i have not yet been described or moderated' % int(pack_id), resource=self)
 
         licenses_url = (reverse('pack-licenses', args=[pack.user.username, pack.id]))
-        return download_sounds(licenses_url, pack)
+        licenses_content = pack.get_attribution()
+        sound_list = pack.sounds.filter(processing_state="OK", moderation_state="OK").select_related('user', 'license')
```

Comment:
I imagine that inside `get_attribution` we already do this query `pack.sounds.filter(processing_state="OK", moderation_state="OK").select_related('user', 'license')` so we are doing it twice. We should improve that. Maybe `get_attribution` could get an optional argument `sound_qs` (`qs`stands for queryset), and when the argument is provided, it does not do the query inside `get_attribution` and it uses the provided queryset. Simlar logic should be applied for the bookmarks case. Do you think it makes sense?